src/cms/templates/icon_widget.html

```diff
@@ -0,0 +1,34 @@
+{% load i18n %}
+{% load form_icon_filters %}
+
+<div class="w-full">
+    {% with icon=widget.value.url label=widget.attrs.label %}
+    <div id="icon_preview" class="{% if not icon %}hidden{% endif %} w-full">
+        <img class="h-40" src="{{ icon }}"/>
```

Comment:
By adding this, the image will be centered :)
```suggestion
        <img class="h-40 m-auto" src="{{ icon }}"/>
```

Reply:
Alright, I centered the image. :+1: